extensions/blocks/donations/attributes.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * WordPress dependencies
+ */
+// eslint-disable-next-line wpcalypso/import-docblock
```

Comment:
I think this (and the ones in other files) can be removed now.

Reply:
Unfortunately we still need it, since JP eslint extends `wpcaypso/react`:

https://github.com/Automattic/jetpack/blob/96d6172619dc7caca1c95d617f4fe5ac766c0158/.eslintrc.js#L4

<img width="733" alt="Screen Shot 2020-07-23 at 11 03 36" src="https://user-images.githubusercontent.com/1233880/88269376-34d92300-ccd4-11ea-84ea-f25a7fd06243.png">

Reply:
Ah, right, I hadn't noticed what that rule was about, my bad!

That's a bit of an odd rule, I wonder if it could be improved so it doesn't trigger when there are no external or internal dependencies to import. I opened https://github.com/Automattic/wp-calypso/issues/44387 and https://github.com/Automattic/wp-calypso/pull/44388 to discuss this.

Reply:
We can now remove the rule exceptions: #16566.